libraries/ESP8266WebServer/examples/Filters/Filters.ino

```diff
@@ -0,0 +1,101 @@
+#include <WiFi.h>
```

Comment:
```suggestion
#include <ESP8266WiFi.h>
```
?

We do have wrapper headers, but for some reason IDE does not like them :/
https://github.com/esp8266/Arduino/blob/master/libraries/ESP8266WiFi/src/WiFi.h

Maybe order, maybe 1.x vs. 2.x